include/odri_control_interface/joint_modules.hpp

```diff
@@ -64,28 +63,26 @@ class JointModules
     std::ostream& msg_out_ = std::cout;
 
 public:
-    JointModules(
-        MasterBoardInterface* robot_if,
-        RefVectorXl motor_numbers,
-        double motor_constants,
-        double gear_ratios,
-        double max_currents,
-        RefVectorXb reverse_polarities,
-        RefVectorXd lower_joint_limits,
-        RefVectorXd upper_joint_limits,
-        double max_joint_velocities,
-        double safety_damping
-    );
+    JointModules(const std::shared_ptr<MasterBoardInterface>& robot_if,
+                 ConstRefVectorXi motor_numbers,
```

Comment:
Did you check if the python bindings work? I had troubles when I wanted to use the integers before as it looks like the python binding was expecting long as type. That's why I implemented the motor_numbers as `RefVectorX[l]` before.

Reply:
Ok you should try it and let me know if the python bindings are working or not.
If not I will provide a setter/accessor for `VectorXl` in the python bindings

Reply:
The yaml parsing goes well and the objects are returned properly in python at least.
I am checking the construction of the robot by hand.